notebooks/concepts/Specviz from splot concept notebook.ipynb

```diff
@@ -0,0 +1,3636 @@
+    "\n",
+    "*Description*:\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### p -- define a linear wavelength scale\n",
+    "\n",
+    "*Description*:\n",
+    "\n",
+    "Define  a  linear  wavelength  scale.  The user is queried for a\n",
+    "        starting  wavelength  and  an  ending  wavelength.   If   either \n",
+    "        (though  not  both)  are  specified  as  INDEF  a  dispersion is\n",
+    "        queried for and used  to  compute  an  endpoint.   A  wavelength\n",
+    "        scale  set this way will be used for other spectra which are not\n",
+    "        dispersion corrected.\n",
+    "    "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### :units -- Change the coordinate units in the plot.\n",
+    "\n",
+    "*Description*:\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Options: units\n",
+    "\n",
+    "*Description*:\n",
+    "\n",
+    "\n",
+    "        Dispersion coordinate units for the plot.  If the  spectra  have\n",
+    "        known  units,  currently  this is generally Angstroms, the units\n",
+    "        may be converted to other units for  plotting  as  specified  by\n",
+    "        this  task  parameter.  If this parameter is the null string and\n",
+    "        the  world  coordinate  system  attribute   \"units_display\"   is \n",
+    "        defined  then  that  will be used.  If both this task parameters\n",
+    "        and \"units_display\" are not given then the  spectrum  dispersion\n",
+    "        units   will   be   used.    The   units  may  also  be  changed \n",
+    "        interactively.  See the units section of the \u000epackage\u000f help for  a\n",
+    "        further description and available units.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### l -- convert to f-lambda\n",
+    "\n",
+    "*Description*:\n",
+    "\n",
+    "Convert to flux per unit wavelength (f-lambda). The spectrum  is\n",
+    "        assumed  to  be  flux  calibrated  in  flux  per  unit frequency\n",
+    "        (f-nu).  See also 'n'."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### n -- convert to fnu\n",
+    "\n",
+    "*Description*:\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Redshifting, line lists, and related spectral coordinate shifts "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There should be a redshift slider (optional?) somewhere in the specviz UI.  This slider should update the redshift/radial_velocity recorded in the spectrum's `.spectral_axis.redshift`/.spectral_axis.radial_velocity` attributes.\n",
+    "\n",
+    "1. In the notebook these can be accessed directly as informational items:"
+   ]
+  },
```

Comment:
There should also be a capability in the viewer to switch between observed and rest (de-redshifted) wavelength, without changing the redshift of the spec1D object.

Reply:
Good point! Will add.